openioc2stix/translate.py

```diff
@@ -1,13 +1,28 @@
+# Copyright (c) 2015, The MITRE Corporation. All rights reserved.
+# See LICENSE.txt for complete terms.
+"""
+Internal module dedicated to translating observables and indicators
+as well as translating OpenIOC to cybOX and STIX.
+"""
 import logging
 
 import cybox.utils
 from cybox.core import Observables, Observable, ObservableComposition
+from stix.common import InformationSource
+from stix.indicator import Indicator
+from stix.core import STIXPackage, STIXHeader
+from stix.common.vocabs import PackageIntent
+
+# python-cybox
+from cybox.common import ToolInformationList, ToolInformation
 
 from . import openioc
 from . import objectify
 from . import xml
 from . import utils
 
+# internal
+from openioc2stix.version import __version__
```

Comment:
Let's make it a relative import since we're inside the `openioc2stix` package (and the imports above it are relative). Move it up with the group above it.